features/validate_image.feature

```diff
@@ -236,3 +244,112 @@ Feature: evaluate enterprise contract
       ]
     }
     """
+    #
+    # In this example the result is the same but in this example the there's only one "source
+    # group" which means the conftest evaluator is run just once with the three sources fetched
+    Given policy configuration named "ec-policy-variation" with specification
```

Comment:
Usually one wouldn't do multiple Given-When-Then sets within the same scenario, breaks the single responsibility and makes testing the individual cases harder. This would be a good case for using a [Rule](https://cucumber.io/docs/gherkin/reference/#rule) instead, and having the setup of the git repositories in a `Background`.